01-gabbar/03-stakater-nordmart-review-ui-bg-blue/01-dev/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: review-web-blue
+description: A Helm chart for Kubernetes
+dependencies:
+  - name: stakater-nordmart-review-ui
+    version: "1.0.14"
```

Comment:
@faizanahmad055 we don't need double quotes they are unnecessary; applies to all files